Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -2530,6 +2550,34 @@ JS::ThrowCompletionOr<JS::NonnullGCPtr<JS::Object>> @constructor_class@::constru
         }
 
         constructor_generator.append(R"~~~(
+    // 7. Set instance.[[Prototype]] to prototype.
+    VERIFY(prototype.is_object());
+    MUST(impl->internal_set_prototype_of(&prototype.as_object()));
```

Comment:
Hmm, is this correct? We are invoking `[[SetPrototypeOf]]` rather than directly setting `[[Prototype]]`.

To set `[[Prototype]]`, we would do:
```cpp
impl->set_prototype(&prototype.as_object());
```

Reply:
TBH, it's coped as-is from the generate_html_constructor code. I think this is correct though, because we're basically replacing `impl.__proto__` with the [[Prototype]] of newTarget.